site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -36,6 +36,18 @@ The same could be run for `alpha.swagger.json` and `beta.swagger.json` as requir
 
 You can read more about OpenAPI/Swagger code generation in their [Command Line Tool Documentation](https://swagger.io/docs/open-source-tools/swagger-codegen/)
 
+## SDK Functionality
+
+| Area      | Action                                                 | Implemented |
+|-----------|--------------------------------------------------------|-------------|
+| Counters  | GET /v1alpha1/counters/{name}                          | ✔️          |
```

Comment:
This part looks good to me. Do we want to add rows for other funcations like the other SDKs?

Reply:
I have the same question 🙋‍♀️

@igooch Looking forward to your inputs

Reply:
I'm not entirely sure what Mark envisioned here, but I would start similarly to the "Action" column like we have in https://agones.dev/site/docs/guides/client-sdks/go/. Then below we can include the Path, Method, Body for each action. The actions for Counters are`GetCounter` `UpdateCounter` and for Lists `GetList` `UpdateList` `AddListValue` `RemoveListValue`. 

I would also include the existing actions "Ready", "Health" etc. that are named below in this doc. (The generated client API is at agones/test/sdk/restapi/swagger/api_sdk.go if you want to see the method names.)